Orange/widgets/data/owsql.py

```diff
@@ -20,15 +22,15 @@
 MAX_DL_LIMIT = 1000000
 
 
-class TableModel(PyListModel):
+class TableModels(PyListModel):
     def data(self, index, role=Qt.DisplayRole):
         row = index.row()
         if role == Qt.DisplayRole:
             return str(self[row])
         return super().data(index, role)
 
 
-class BackendModel(PyListModel):
+class BackendModels(PyListModel):
```

Comment:
[Model](http://doc.qt.io/qt-5/model-view-programming.html) is a part of the Qt's Model/View paradigm and should remain singular.